src/main/java/auto_batcher/Invoker.java

```diff
@@ -0,0 +1,38 @@
+package org.txstate.auto_batcher;
+
+import java.util.*;
+import java.io.*;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class Invoker
```

Comment:
the name is misleading. it is not invoking anything.
it is simply retrieving name of `.json` files in a a folder.

i would have this class something like `StudentRepository` or `StudentDatabase`